CHANGELOG_en-GB.md

```diff
@@ -1,3 +1,11 @@
+# 2.0.2
+* Customers are now being updated in the Unzer Insight Board
+* Fixed backwards compatibility to Shopware 6.3 and lower for SEPA payment methods
+* Fixed the maximum birthday year
+* Fixed the order list override so other plugins are also able to manipulate the list
+* Compatibiity with showpare 6.4.3.0 ensured
```

Comment:
Two typos: `Compatibility` and `Shopware`

Reply:
Good catch. I've fixed these. Thanks :)